### ISSUE TITLE
guild color name

### ISSUE THREAD

**Issue by Macagna01** [Backupiseasy/ThreatPlates#504]
hello, wanted to change guild color name for guildmate, in widget social
original is blue 3ca8ff, but every time i try to change the color it turn back in blue, cant swing.
any idea?

**Comment by Backupiseasy** [comment]
Yes, every time I run into this issue myself I am confused. For me, it was the alpha/transparency value that was set to 0 or 1 which resulted in the actual color not being shown, but only black was selected.

I am not sure if it's the same for you, but when that happens, the color selector first looks like this:

![image](https://github.com/Backupiseasy/ThreatPlates/assets/20885351/8214b958-e83b-4608-904f-cb67edbff06c)

You can see I selected red as color, but the color is still shown as black in the color square on the right. 

Then I change the alpha/transparency value and it looks like this:

![image](https://github.com/Backupiseasy/ThreatPlates/assets/20885351/d3312b6d-b339-4d7c-a786-607b17947984)

**Comment by Macagna01** [comment]
![guild color](https://github.com/Backupiseasy/ThreatPlates/assets/159602595/c050e3e1-f308-411d-ad10-a38ba33bb97a)

**Comment by Backupiseasy** [comment]
There was a update to an embedded library (Ace3) that had some changes to the color picker. Here's an update of TP with this new library version: https://drive.google.com/file/d/1wzLwt6qmLsHU7US2RW6fzaAFkn2EhzqT/view?usp=sharing

You can install this library separately which might prevent the error for me if it's related to this. But I am not sure.

**Comment by Macagna01** [comment]
thanks for new update.
my issue still here, even with direct change guildmatecolor in "constants.lua" , in game all character guild name have the same default color.
cant have my guildmate "guild" in pink and other people in common blue.
I had  missing something :/ ...

**Comment by Backupiseasy** [comment]
Ok, I will think about it some more what could cause this. From the look of it, are you using ElvUI?

**Comment by Macagna01** [comment]
Yes i do

**Comment by Backupiseasy** [comment]
Do you color healthbars by health or by class? If coloring by health is used, coloring by guild is disabled.

**Comment by Macagna01** [comment]
I use colored by class.
Actually all guilds tags are in the same color in headline view.
I can’t find how see my guildmembers in another color.

**Comment by Backupiseasy** [comment]
Ah, you said guilt tags. You do not mean the healthbar color or the name color, but the color of the icon that shouls a guild mate. Is that correct? It it is like this ... the color of this icon cannot be changed. Only the color of healthbar and name.

**Comment by Macagna01** [comment]
Guild name
